interpreter/value_accountcapabilitycontroller.go

```diff
@@ -241,6 +235,22 @@ func (v *AccountCapabilityControllerValue) GetMember(context MemberAccessibleCon
 
 	case sema.AccountCapabilityControllerTypeCapabilityFieldName:
 		return v.GetCapability(context)
+	}
+
+	return context.GetMethod(v, name, locationRange)
+}
+
+func (v *AccountCapabilityControllerValue) GetMethod(
+	context MemberAccessibleContext,
+	locationRange LocationRange,
+	name string,
+) FunctionValue {
+	switch name {
```

Comment:
Not sure if this is needed, but better to have this here, like in `GetMember`, in case `GetMethod` is invoked outside of `GetMember`:

```suggestion
	// NOTE: check if controller is already deleted
	v.checkDeleted()

	switch name {
```